Resources/templates/responsive/invest/select_reward.php

```diff
@@ -14,25 +14,21 @@
 
                 <?= $this->supply('sub-header') ?>
 
-                <?= $this->insert('invest/partials/reward_box_resign') ?>
+                <form class="form-horizontal" role="form" method="GET" action="/invest/<?= $this->project->id ?>/payment">
+                    <?= $this->insert('invest/partials/reward_box_resign') ?>
 
-                <?php foreach($this->rewards as $reward_item): ?>
-
-                    <?php //check if show the exhausted rewards ?>
-                    <?php if($reward_item->available()||!$this->project::hideExhaustedRewards($this->project->id)||!$this->project->inCampaign()): ?>
-
-                        <?= $this->insert('invest/partials/reward_box', ['reward_item' => $reward_item]) ?>
-
-                    <?php endif; ?>
-
-                <?php endforeach ?>
+                    <?php foreach($this->rewards as $reward_item):
+                        if ($reward_item->archived) continue;
```

Comment:
In my opinion, archived rewards are also unavailable rewards. Can't we update the `$reward_item->available()` method to also check if the reward is archived? That way we can also remove this double check and further ones in the code.

Reply:
The contract for the `available` function is related to the relation between the reward units available from the project owner and the number of times the reward has been taken by a donation. Therefore i don't think we can change it for another functionallity. We can still unify it with the following if statement.